lib/utils.coffee

```diff
@@ -19,6 +19,8 @@ semver = require('semver')
 fs = Promise.promisifyAll(require('fs'))
 resin = require('resin-sdk-preconfigured')
 
+RESINOS_VERSION_REGEX = /v?\d+\.\d+\.\d+(\.rev\d+)?((\-|\+).+)?/
```

Comment:
Can we expose the version validation logic from the SDK? Now that we have our own version format I'd like this to be reused from a single source.